src/rgw/CMakeLists.txt

```diff
@@ -129,6 +129,7 @@ add_library(rgw_common OBJECT ${librgw_common_srcs})
 
 target_include_directories(rgw_common SYSTEM PUBLIC "services")
 target_include_directories(rgw_common PUBLIC "${CMAKE_SOURCE_DIR}/src/dmclock/support/src")
+target_include_directories(rgw_common PUBLIC "${CMAKE_SOURCE_DIR}/src/fmt/include")
```

Comment:
it's harmless, but this presumably has no effect in the backport

Reply:
> it's harmless, but this presumably has no effect in the backport

fixed anyway